Rakefile

```diff
@@ -27,6 +22,7 @@ end
 namespace :test do
   desc 'Run all development tests located in the test directory'
   Rake::TestTask.new :dev do |task|
+    task.options = RakeArguments.task_options
```

Comment:
Hmm... the `--` generally signals that options are done being passed to the command being ran, and used for some command that the first command is utilizing.  The name task_options may be appropriate here.  Unsure what to do here.